### PR TITLE
Link to the Onigurama docs

### PR DESCRIPTION
**Pull request by strugee**
Not sure if the formatting here is right - if this isn't right let me know and I can amend locally.

### CODE REVIEW

File: docs/content/manual/manual.yml [hunk 1]
```diff
@@ -2277,9 +2277,10 @@ sections:
   - title: Regular expressions (PCRE)
     body: |
 
-      jq uses the Oniguruma regular expression library, as do php,
-      ruby, TextMate, Sublime Text, etc, so the description here
-      will focus on jq specifics.
+      jq uses the
+      [Oniguruma regular expression library](https://github.com/kkos/oniguruma/blob/master/doc/RE)
+      , as do php, ruby, TextMate, Sublime Text, etc, so the
```
Comment (itchyny) [inline docs/content/manual/manual.yml]: It's preferable to place the comma at the end of the previous line to omit spaces before it.

Comment (strugee) [reply docs/content/manual/manual.yml]: Not quite sure how that happened - amended

**Review by itchyny — APPROVED** [review]
LGTM, thank you!

**Comment by strugee** [comment]
No problem! @itchyny do you need anything else from me?

**Pull request merged**